Automattic/jetpack
# AI Assistant: Send array of messages instead of string prompt

Adjusts the frontend to D112325-code

## Proposed changes:

* Changes the prompt from a single string to an array of messages
* Removes the `summarize` option when generating a title
* Adds multiple modifications to the prompts, both for better represent the array structure and to get better results overall

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add some AI Assistant blocks and play with them
* Try to use them as you would in a "normal" flow
* Try to stress them with different languages and commands

You can check the prompts in the console by using `localStorage.setItem( 'debug', 'jetpack-ai-assistant:prompt' )` or `localStorage.setItem( 'debug', '*' )`.
